src/solvers/mp/RCPricer.cpp

```diff
@@ -33,26 +33,26 @@ RotationPricer::RotationPricer(MasterProblem* master, const char* name, SolverPa
 }
 
 /* Destructs the pricer object. */
-RotationPricer::~RotationPricer() {
-	for(pair<const Contract*, SubProblem*> p: subProblems_)
+RCPricer::~RCPricer() {
+	for(auto& p: subProblems_)
 		delete p.second;
 }
 
-void RotationPricer::initPricerParameters(SolverParam param){
+void RCPricer::initPricerParameters(const SolverParam& param){
 	// Here: doing a little check to be sure that secondchance is activated only when the parameters are different...
 	withSecondchance_ = param.sp_withsecondchance_ && (param.sp_default_strategy_ != param.sp_secondchance_strategy_);
-	nbMaxRotationsToAdd_ = param.sp_nbrotationspernurse_;
+	nbMaxColumnsToAdd_ = param.sp_nbrotationspernurse_;
 	nbSubProblemsToSolve_ = param.sp_nbnursestoprice_;
 	defaultSubprobemStrategy_ = param.sp_default_strategy_;
 	secondchanceSubproblemStrategy_ = param.sp_secondchance_strategy_;
-
+    shortSubproblem_ = param.sp_short_;
 	currentSubproblemStrategy_ = defaultSubprobemStrategy_;
 }
 
 /******************************************************
  * Perform pricing
  ******************************************************/
-vector<MyVar*> RotationPricer::pricing(double bound, bool before_fathom){
+vector<MyVar*> RCPricer::pricing(double bound, bool before_fathom) {
```

Comment:
Reading this function made me think of how columns are managed by BCP. Do you know if its stores everything in a big pool where it picks those that are relevant to the current node everytime it solves a MP? 
Would there be any value in trying to price columns that were already generated for different branching nodes? Or for other nurses? 
Is this worth opening an issue according to you?

Reply:
I know that BCP deletes some columns, but that's all. I think it's not urgent.